controller/gen/apis/serviceprofile/v1alpha1/types.go

```diff
@@ -27,8 +28,9 @@ type ServiceProfile struct {
 
 // ServiceProfileSpec specifies a ServiceProfile resource.
 type ServiceProfileSpec struct {
-	Routes      []*RouteSpec `json:"routes"`
-	RetryBudget *RetryBudget `json:"retryBudget,omitempty"`
+	Routes       []*RouteSpec   `json:"routes"`
+	RetryBudget  *RetryBudget   `json:"retryBudget,omitempty"`
+	DstOverrides []*WeightedDst `json:"dstOverrides,omitempty"`
```

Comment:
Doesn't this need a corresponding change to the ServiceProfile CRD?

Reply:
... well I guess not given we don't care about the bare-bones validation the CRD provides